ember-engines-router-service/package.json

```diff
@@ -59,6 +56,7 @@
     "@types/ember__routing": "^4.0.22",
     "babel-plugin-ember-template-compilation": "^2.2.5",
     "concurrently": "^8.2.2",
+    "ember-source": "^6.4.0",
```

Comment:
Isn't it necessary to keep covering old Ember versions as we were doing before?

Reply:
ember-try does this for us